src/Open3D/Utility/Eigen.cpp

```diff
@@ -278,5 +278,26 @@ template std::tuple<Eigen::Matrix6d, Eigen::Vector6d, double> ComputeJTJandJTr(
         int iteration_num, bool verbose);
 // clang-format on
 
+Eigen::Matrix3d RotationMatrixX(double radians) {
+    Eigen::Matrix3d rot;
```

Comment:
Eigen already provide the code for this see https://github.com/openMVG/openMVG/blob/master/src/openMVG/numeric/numeric.cpp#L46